packages/web-runtime/src/components/VersionCheck.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div v-if="isLoading" class="version-check-loading flex items-center">
+    <span v-text="$gettext('Checking for updates')" />
+    <oc-spinner class="ml-1" size="xsmall" />
+  </div>
+  <div v-else>
+    <div v-if="!updateAvailable" class="version-check-no-updates flex items-center">
+      <span v-text="$gettext('Up to date')" />
+      <oc-icon class="ml-0.5" name="checkbox-circle" size="xsmall" fill-type="line" />
+    </div>
+    <oc-button
+      v-else
+      class="version-check-update text-role-on-surface-variant"
+      size="small"
+      type="router-link"
+      :href="updateData.url"
+      target="_blank"
+      gap-size="small"
+      appearance="raw"
+    >
+      <span
+        class="text-xs"
+        v-text="$gettext('Version %{version} available', { version: updateData.current_version })"
+      />
+      <oc-icon name="refresh" size="xsmall" fill-type="line" />
+    </oc-button>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { onMounted, computed, ref } from 'vue'
+import { useTask } from 'vue-concurrency'
+import { useGettext } from 'vue3-gettext'
+import semver from 'semver'
+import { useCapabilityStore, useClientService, useConfigStore } from '@opencloud-eu/web-pkg/src'
+
+export interface UpdateChannelData {
+  current_version: string
+  url: string
+}
+
+export type UpdateChannelName = 'rolling' | 'stable' | `lts${string}`
```

Comment:
Could you please remove the `lts` type here and only keep `rolling` and `stable` for now? We can introduce lts types when we have an LTS version.